### ISSUE TITLE
executor failed running [/bin/sh -c apk add --no-cache python g++ make]

### ISSUE THREAD

**Issue by quinncomendant** [docker/getting-started#214]
When trying to build, the `RUN apk add --no-cache python g++ make` step fails:

```
#> time docker build -t docker101tutorial .
[+] Building 4.9s (13/24)                                                                                                                                                          
 => [internal] load build definition from Dockerfile                                                                                                                          0.0s
 => => transferring dockerfile: 37B                                                                                                                                           0.0s
 => [internal] load .dockerignore                                                                                                                                             0.0s
 => => transferring context: 34B                                                                                                                                              0.0s
 => [internal] load metadata for docker.io/library/nginx:alpine                                                                                                               1.8s
 => [internal] load metadata for docker.io/library/python:alpine                                                                                                              0.0s
 => [internal] load metadata for docker.io/library/node:12-alpine                                                                                                             1.7s
 => CACHED [app-base 1/8] FROM docker.io/library/node:12-alpine@sha256:1ecf7789a48e604a841409e65cf2e172d5a014ecd5e60689137d785d2613e9b0                                       0.0s
 => CACHED [stage-5 1/3] FROM docker.io/library/nginx:alpine@sha256:686aac2769fd6e7bab67663fd38750c135b72d993d0bb0a942ab02ef647fc9c3                                          0.0s
 => [base 1/4] FROM docker.io/library/python:alpine                                                                                                                           0.0s
 => [internal] load build context                                                                                                                                             0.1s
 => => transferring context: 7.72kB                                                                                                                                           0.1s
 => ERROR [app-base 2/8] RUN apk add --no-cache python g++ make                                                                                                               2.8s
 => CACHED [base 2/4] WORKDIR /app                                                                                                                                            0.0s
 => CACHED [base 3/4] COPY requirements.txt .                                                                                                                                 0.0s
 => CANCELED [base 4/4] RUN pip install -r requirements.txt                                                                                                                   2.9s
------                                                                                                                                                                             
 > [app-base 2/8] RUN apk add --no-cache python g++ make:                                                                                                                          
#8 0.737 fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz                                                                                             
#8 1.677 fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
#8 2.633 ERROR: unable to select packages:
#8 2.746   python (no such package):
#8 2.746     required by: world[python]
------
executor failed running [/bin/sh -c apk add --no-cache python g++ make]: exit code: 1
```

Steps to reproduce:

1. `git clone https://github.com/docker/getting-started.git`
2. `cd getting-started/`
3. `docker build -t docker101tutorial .`

I'm using Docker Desktop 4.1.1 on a 2015 MacBook Pro with Mojave 10.14.6.

**Comment by chenrc98** [comment]
`RUN apk add --no-cache python2 g++ make'
Change python to python2 may fix this issue

**Comment by Jerem-Mar** [comment]
With python3 also works

**Comment by StefanScherer** [comment]
Thanks @quinncomendant for the report. Indeed that package `python` was removed and now we would have to use `python3` instead.
To make the tutorial a bit more stable I've merged #212 to remove this line. The tutorial should work for you again.
I'm working on a fix for Apple silicon/M1 users now.

**Comment by quinncomendant** [comment]
It works now. Thanks!
